Basic-Car-Maintenance/Shared/Onboarding/Views/WelcomeViewAddVehicle.swift

```diff
@@ -9,99 +9,177 @@ import SwiftUI
             }
-            .frame(maxWidth: .infinity, alignment: .leading)
-            .padding(.horizontal, 15)
-            
-            Text("You can edit more data about the vehicle in the 'Settings' tab.")
-                .foregroundStyle(.gray)
-                .padding(.horizontal, 25)
+        }
+        .font(.largeTitle.bold())
+        .multilineTextAlignment(.center)
+        .padding(.top, 65)
+        .padding(.bottom, 15)
+    }
+    
+    private var vehicleDetailsView: some View {
+        VStack {
+            Image(systemName: "car.side.lock.open")
+                .font(.system(size: 45))
+                .foregroundStyle(Color("basicGreen"))
             
-            Spacer(minLength: 10)
+            List {
+                vehicleDetailRow(title: "Name", text: $vehicleName, alert: $validationAlertName)
+                vehicleDetailRow(title: "Make", text: $vehicleMake, alert: $validationAlertMake)
+                vehicleDetailRow(title: "Model", text: $vehicleModel, alert: $validationAlertModel)
+            }
+            .frame(maxHeight: 200)
+            .scrollContentBackground(.hidden)
+        }
+        .padding(.horizontal, 15)
+    }
+    
+    private func vehicleDetailRow(title: String, text: Binding<String>, alert: Binding<Bool>) -> some View {
+        LabeledContent {
+            TextField("Vehicle \(title)", text: text)
+                .alert(
+                    "Vehicle \(title) cannot be empty! 🚗",
+                    isPresented: alert
+                ) {
+                    Button("OK", role: .cancel) {}
+                }
+                .frame(width: 200)
+        } label: {
+            HStack {
+                Text(title)
+                Spacer()
+            }
+        }
+        .showClearButton(text)
+    }
+    
+    private var bottomText: some View {
+        Text("You can edit more data about the vehicle in the 'Settings' tab.")
+            .foregroundStyle(.gray)
+            .padding(.horizontal, 25)
+    }
+    
+    private var addVehicleButton: some View {
+        Button {
+            addVehicle()
+        } label: {
+            Text("Add Vehicle")
+                .fontWeight(.bold)
+                .foregroundStyle(.white)
+                .frame(maxWidth: .infinity)
+                .padding(.vertical, 14)
+                .background(Color("basicGreen").gradient, in: .rect(cornerRadius: 12))
+                .contentShape(.rect)
+        }
+        .padding(15)
+        .padding(.horizontal, 15)
+        .alert(
+            Text("Congratulations! 🎉 🚙"),
+            isPresented: $onboardingViewCompletedAlert
+        ) {
+            Button("OK", role: .cancel, action: {
+                isFirstTime = false
+            })
+        } message: {
+            Text(
+                "From now on, next screen will be the main app screen.",
+                comment: "Shows popup when completed Onboarding tutorial"
+            )
+        }
```

Comment:
I would prefer to not have this alert, or if you do only do something like "Vehicle name" added

Reply:
Ok, I will do it 👍 Will be "Vehicle name" added